frappe_slack_connector/db/timesheet.py

```diff
@@ -4,6 +4,70 @@
 from frappe_slack_connector.db.employee import get_employee_from_user
 
 
+def get_user_projects(user: str, limit: int | None = 90) -> list:
+    """
+    Get the projects for the given user
+    """
+    projects = frappe.get_all(
```

Comment:
Add pagination hear also

Reply:
added `limit` in `get_list` in both calls